ui/src/app/base/components/StatusBar/StatusBar.tsx

```diff
@@ -60,36 +63,32 @@ export const StatusBar = (): JSX.Element | null => {
     return null;
   }
 
-  let status: ReactNode = "";
-  if (isDeployedWithHardwareSync(activeMachine)) {
+  let status: ReactNode;
+  if (isMachineDetails(activeMachine)) {
+    const statuses = [activeMachine.fqdn];
+    if (isDeployedWithHardwareSync(activeMachine)) {
+      statuses.push(
+        `Last synced: ${getSyncStatusString(activeMachine.last_sync)}`
+      );
+      statuses.push(
+        `Next sync: ${getSyncStatusString(activeMachine.next_sync)}`
+      );
+    } else {
+      statuses.push(getLastCommissionedString(activeMachine));
+    }
     status = (
-      <ul className="p-inline-list u-no-margin--bottom">
-        <li className="p-inline-list__item">
-          <strong>{activeMachine.fqdn}</strong>
-        </li>
-        <li className="p-inline-list__item">
-          Last synced: {getSyncStatusString(activeMachine.last_sync)}
-        </li>
-        <li className="p-inline-list__item">
-          Next sync: {getSyncStatusString(activeMachine.next_sync)}
-        </li>
+      <ul className="p-inline-list u-flex--wrap u-no-margin--bottom">
+        {statuses.map((status, i) => (
+          <li className="p-inline-list__item" key={status}>
+            {i === 0 ? <strong>{status}</strong> : status}
+          </li>
+        ))}
```

Comment:
Previous implementation was arguably easier to comprehend at a glance. You could try and inline the first `li` element containing the machine name Instead of relying on the index to apply different style.

Reply:
That's fair, relying on the index is a bit iffy. Maybe something to go over once some other contextual stuff is added